elliottlib/cli/find_builds_cli.py

```diff
@@ -127,6 +132,20 @@ def find_builds_cli(runtime, advisory, default_advisory_type, builds, kind, from
             click.echo(' ' + b.nvr)
 
 
+def _get_product_version(nvr, product_version_map, brew_session=koji.ClientSession(constants.BREW_HUB)):
+    product_version = ""
+    for tag in brew_session.listTags(nvr):
+        tag_name = tag.get('name')
+        product_version = product_version_map.get(tag_name, "")
+        if product_version:
+            return product_version
```

Comment:
It is possible that one brew build is associated with multiple candidate tags.

Reply:
this function is no longer needed, I've already removed it.